tests/helpers.py

```diff
@@ -710,6 +723,25 @@ def get_cephfs_name():
     return result['items'][0].get('metadata').get('name')
 
 
+def pull_images(image_name):
+    """
+    Function to pull images on all nodes
+    Args:
+        image_name (str): Name of the container image to be pulled
+
+    Returns:
+        bool: True if deletion of image pull is successful
+    """
+    nodes = get_worker_nodes()
+    for worker_node in nodes:
+        node_obj = ocp.OCP(kind=constants.NODE)
```

Comment:
Can we use get_node_objs() function here ?

Reply:
Thanks for the pointer. I've used `get_node_objs` now.